<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_Departments</class>
 <widget class="QDialog" name="Edit_Departments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование отделов</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Отдел</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QToolButton" name="editUserButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/Edit-find-replace.svg</normaloff>:/icons/Edit-find-replace.svg</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QToolButton" name="delUserButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/List-remove.svg</normaloff>:/icons/List-remove.svg</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Пользователи</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="addUserButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/List-add.svg</normaloff>:/icons/List-add.svg</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QToolButton" name="editDepartmentButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/Edit-find-replace.svg</normaloff>:/icons/Edit-find-replace.svg</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Edit_Departments</receiver>
   <slot>changeDepartments(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addUserButton</sender>
   <signal>clicked()</signal>
   <receiver>Edit_Departments</receiver>
   <slot>addUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editUserButton</sender>
   <signal>clicked()</signal>
   <receiver>Edit_Departments</receiver>
   <slot>editUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delUserButton</sender>
   <signal>clicked()</signal>
   <receiver>Edit_Departments</receiver>
   <slot>delUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editDepartmentButton</sender>
   <signal>clicked()</signal>
   <receiver>Edit_Departments</receiver>
   <slot>editDepartments()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeDepartments(int)</slot>
  <slot>addUser()</slot>
  <slot>editUser()</slot>
  <slot>delUser()</slot>
  <slot>editDepartments()</slot>
 </slots>
</ui>
